tests/torch/quantization/test_functions.py

```diff
@@ -96,39 +47,51 @@ def _seed():
     np.random.seed(42)
 
 
-def generate_one_channel_input(input_low, input_range, min_adj, quant_len, ch_idx, input_size,
-                               bits, get_deviation, min_deviation):
-    if np.prod(min_adj.shape) == 1:
-        min_adj_ch, input_low_ch, input_range_ch, quant_len_ch =\
-            map(lambda x: x.squeeze(), [min_adj, input_low, input_range, quant_len])
+RQ = ReferenceQuantize(backend_type=ReferenceBackendType.NUMPY)
+
+
+def generate_one_channel_input(input_low: np.ndarray,
+                               input_range: np.ndarray,
+                               ch_idx: Optional[List[int]],
+                               input_size: List[int],
+                               bits: int,
+                               get_deviation: Callable[[], int],
+                               min_deviation: float):
+    if np.prod(input_low.shape) == 1:
+        input_low_ch, input_range_ch = \
+            map(lambda x: x.squeeze(), [input_low, input_range])
     else:
-        min_adj_ch, input_low_ch, input_range_ch, quant_len_ch =\
-            map(lambda x: x[tuple(ch_idx)].squeeze(), [min_adj, input_low, input_range, quant_len])
+        input_low_ch, input_range_ch = \
```

Comment:
Is it equivalent without `min_adj_ch` and `quant_len`? Why was it needed before?

Reply:
No idea, the original code was by @daniil-lyakhov. I reviewed the usage of these variables carefully and saw no reason to keep them, they are equivalent to some arguments already in the scope. Maybe @daniil-lyakhov just forgot to remove them.

Reply:
Yeah, for previous version of the quantize function this `min_adj_ch` was not equal to `-inp_low`. So, this changes are totally valid